packages/editor/src/components/parts/web-service/Permission.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { CollapsableUtil, SelectUtil, render } from 'test-utils';
+import { Permission } from './Permission';
+
+describe('Permission', () => {
+  test('data', async () => {
+    render(<Permission />, { wrapperProps: { data: { config: { permission: { error: '>> Ignore Exception', role: 'Test' } } } } });
+    await CollapsableUtil.assertOpen('Permission');

```

Comment:
would be nice if you also can somewhere test that this is closed on default data